src/main/resources/templates/kotlin-lang/kotlinClassGraphqlResponseProjection.ftl

```diff
@@ -26,10 +26,12 @@ import java.util.Objects
 </#list>
 open class ${className} : GraphQLResponseProjection() {
 
-<#if fields?has_content>
-    override fun `all$`(): ${className} = `all$`(${responseProjectionMaxDepth})
+<#if fields?has_content && generateAllMethodInProjection>
+    var projectionDepthOnFields: MutableMap<String, Int> = HashMap<String, Int>()
```

Comment:
Must use `MutableMap`, otherwise, it cannot be modified.
```kotlin

    val projectionDepthOnFields: MutableMap<String, Int> = mutableMapOf()
    projectionDepthOnFields.put(key, value)
```

Reply:
please use this way:
```
    val projectionDepthOnFields2: MutableMap<String, Int> = mutableMapOf()
    projectionDepthOnFields2["key"] = value
```

Reply:
Good improvement thank you! I changed them.